rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java

```diff
@@ -88,7 +91,7 @@ void whenOldAndNewFederationArePresentReturnOldFederationByActivationAge() {
         when(provider.getNewFederation()).thenReturn(newFederation);
         when(provider.getOldFederation()).thenReturn(oldFederation);
         when(executionBlock.getNumber()).thenReturn(80L);
-        when(bridgeConstants.getFederationActivationAge()).thenReturn(10L);
+        when(bridgeConstants.getFederationActivationAge(activations)).thenReturn(10L);
```

Comment:
Should we test this pre and post fork? Same with the test below

Reply:
See: https://github.com/rsksmart/rskj/blob/increase-new-fed-activation-age/rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java#L69-L105

Reply:
Done